projects/rero/ng-core/src/lib/translate/date-translate-pipe.ts

```diff
@@ -34,12 +34,10 @@ export class DateTranslatePipe extends DatePipe implements PipeTransform {
     super(_translateService.currentLanguage);
   }
 
-  transform(
-    value: any,
-    format = 'mediumDate',
-    timezone?: string,
-    locale?: string
-  ): string {
+  transform(value: Date | string | number, format?: string, timezone?: string, locale?: string): string | null;
+  transform(value: null | undefined, format?: string, timezone?: string, locale?: string): null;
+  transform(value: Date | string | number | null | undefined, format?: string, timezone?: string, locale?: string): string | null
```

Comment:
Why 3 times the same function ? Only one needed

Reply:
This is required as this function is inherited from the parent pipe.

Reply:
Thanks for the precision 👍🏻